runtime/moonbase/tests/xcm_mock/parachain.rs

```diff
@@ -680,6 +680,54 @@ impl xcm_transactor::Config for Runtime {
 	type AssetTransactor = AssetTransactors;
 }
 
+parameter_types! {
+	pub const MinimumPeriod: u64 = 1000;
+}
+impl pallet_timestamp::Config for Runtime {
+	type Moment = u64;
+	type OnTimestampSet = ();
+	type MinimumPeriod = MinimumPeriod;
+	type WeightInfo = ();
+}
+
+pub struct FixedGasPrice;
+impl pallet_evm::FeeCalculator for FixedGasPrice {
+	fn min_gas_price() -> U256 {
+		1_000_000_000u128.into()
+	}
+}
+
+pub struct FindAuthorTruncated;
```

Comment:
Same here. Find Author is implemented for (), and since I think that this does not play a role in our tests, we can make it ()